### PR TITLE
Render names in authorization list

### PR DESCRIPTION
**Pull request by bfollington**
Fixes https://github.com/subconsciousnetwork/subconscious/issues/1060

- Fetch name of authorizations
- Render in list view
- Move to `.contextMenu` for revocation and sharing

https://github.com/subconsciousnetwork/subconscious/assets/5009316/ddb7206d-45b1-40bb-9cec-d891a41a55a8

### CODE REVIEW

File: xcode/Subconscious/Shared/Services/Noosphere/Sphere.swift [hunk 1]
```diff
@@ -949,6 +968,34 @@ public actor Sphere: SphereProtocol, SpherePublisherProtocol {
         }
     }
     
+    public func authorizationName(authorization: Authorization) async throws -> String {
```
Comment (gordonbrander) [inline xcode/Subconscious/Shared/Services/Noosphere/Sphere.swift:971]: Let's add a test for this method

**Review by gordonbrander — CHANGES_REQUESTED** [review]
Looks good, just recommend adding one test

File: xcode/Subconscious/Shared/Services/Noosphere/Sphere.swift (hunk 1 above)
Comment (bfollington) [reply xcode/Subconscious/Shared/Services/Noosphere/Sphere.swift:971]: Good call, done!

**Review by gordonbrander — APPROVED** [review]
LGTM

**Pull request merged**